tenant/store.go

```diff
@@ -38,10 +40,11 @@ func (tenantStore *tenantStore) Entities() []interface{} {
 }
 
 type Tenant struct {
-	Id       uint64 `sql:"AUTO_INCREMENT"`
-	Name     string
-	Segments []Segment
-	Seq      uint64
+	ID         uint64 `sql:"AUTO_INCREMENT"`
+	ExternalID string `sql:"not null;unique"`
```

Comment:
Yes I did get to this point yesterday and wanted to create an extra table to map the linking thus we would know exactly which fields are related to external id, but i thought that would go as an separate task, in case it makes sense I will change this behavior and add the mapping for external id here, and we can add separate task for other external id's separately.